lib/simplifyExpression/fractionsSearch/cancelLikeTerms.js

```diff
@@ -287,6 +289,56 @@ function cancelTerms(numerator, denominator) {
     return new CancelOutStatus(numerator, denominator, true);
   }
 
+  // case 4: the numerator is a constant and denominator is a polynomial term
+  // e.g. 2 / 4x -> 1 / 2x
+
+  if (Node.Type.isConstant(numerator) && Node.PolynomialTerm.isPolynomialTerm(denominator)) {
+    const denominatorTerm = new Node.PolynomialTerm(denominator);
+
+    const coeff = denominatorTerm.getCoeffNode();
+    const variable = denominatorTerm.getSymbolNode();
+    const exponent = denominatorTerm.getExponentNode();
+
+    // simplify a constant fraction (e.g 2 / 4)
+    const frac = Node.Creator.operator('/', [numerator, coeff]);
+
+    let newCoeff;
+    const reduceStatus = divideByGCD(frac);
+
+    if (reduceStatus.hasChanged()) {
+      [numerator, newCoeff] = reduceStatus.newNode.args;
+    }
+    else {
+      return new CancelOutStatus(numerator, denominator, false);
+    }
+
+    denominator = Node.Creator.polynomialTerm(variable, exponent, newCoeff);
+
+    return new CancelOutStatus(numerator, denominator, true);
+  }
+
+  // case 5: both numerator and denominator are numbers
+  // e.g. 2 / 4 -> 1 / 2
+  // e.g. (35 * nthRoot (7)) / (5 * nthRoot(5)) -> (7 * nthRoot(7)) / nthRoot(5)
+
+  if (Node.Type.isConstant(numerator) && Node.Type.isConstant(denominator)) {
+    const frac = Node.Creator.operator('/', [numerator, denominator]);
+    const reduceStatus = divideByGCD(frac);
+    if (reduceStatus.hasChanged()) {
```

Comment:
same returning early thing
```
if (!reduceStatus.hasChanged()) {
  return new CancelOutStatus(numerator, denominator, false);
}
... continue as usual, but less indented
```

I care about this because I see the else below then have to look up a bit to see what it's else-ing, this way it's much more clear